showcase/src/assets/translations/en.json

```diff
@@ -236,6 +236,20 @@
         "TITLE": "Stark Typography"
       }
     },
+    "HOMEPAGE": {
+      "MAIN_TITLE": "About Stark",
+      "STARK_DESC_BEFORE_DETAIL": "Stark is an Angular 5+ based front-end framework built on top of Angular Stark features... An awesome reusable build using Webpack, built-in support for state of the art front-end tech. Stark provides main building blocks for accelerating front-end development:",
+      "STARK_DESC_DETAIL": "<ul><li>a solid reusable build based on Webpack</li><li>a starter project inspired by Angular Starter by AngularClass and Google's Web Starterk Kit</li><li>core modules providing useful APIs</li><li>UI modules providing interesting UI components </li><li>a showcase application containing: <ul><li>our living style guide</li><li>a showcase of all components/services along with their API</li><li>our developer guide</li></ul></li></ul>",
+      "STARK_DESC_AFTER_DETAIL": "Stark modules are like LEGO blocks: add what you need, no less, no more. If you don't like our defaults then you may replace/override/ignore anything you fancy.<br> <a href=\"https://github.com/NationalBankBelgium\"><img class=\"github-link\" src=\"/assets/images/logo/github.png\"></a> And even better, the Stark repository is available on the National Bank of Belgium page on Github!<br>Come and have a look at our amazing open sources projects!",
```

Comment:
Can you add these attributes in all the `<a>` links?  
`target="_blank"` so that the link opens on a new tab instead of the current one!
and `rel="noopener noreferrer"` see: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/